ibodata/depth_profile.py

```diff
@@ -1,6 +1,37 @@
+import numpy as np
+
 from beprof.profile import Profile
 
 
 class DepthProfile(Profile):
-    """TODO"""
-    pass
+
+    def fwhm(self):
```

Comment:
Why are you re-implementing it ?
It is a method of a base class:

https://github.com/DataMedSci/beprof/blob/master/beprof/profile.py#L128